.github/workflows/publish-commit-bottles.yml

```diff
@@ -68,14 +73,35 @@ jobs:
         with:
           signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
 
+      - name: Checkout PR branch
+        run: gh pr checkout ${{github.event.inputs.pull_request}}
+        if: inputs.commit_bottles_to_pr_branch
+        env:
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Yep, I've tried `GITHUB_TOKEN`, but it seems to not have the right permissions either.

Reply:
@carlocab Working now. Needed to add `permissions`.